src/api/spec/controllers/status/checks_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe Status::ChecksController, type: :controller do
+  let(:user) { create(:confirmed_user) }
+  let(:project) { create(:project_with_repository) }
+  let(:repository) { project.repositories.first }
+  let(:status_repository_publish) { create(:status_repository_publish, repository: repository) }
+
+  before do
+    login user
+  end
+
+  describe 'GET index' do
+    it 'assigns @checks' do
+      check1 = create(:check, checkable: status_repository_publish)
+      check2 = create(:check, checkable: status_repository_publish)
```

Comment:
Please move this to `let` statements :-)

Reply:
But they are `it` specific.

Reply:
That's always the case.

If you don't want to define them for the whole test file, you can still add it to the `describe 'GET index' do` block.